lib/output.rb

```diff
@@ -0,0 +1,41 @@
+class Output
+
+  def initialize(order, dishes)
+    @order = order
+    @dishes = dishes
+    @line_width = 80
+  end
+
+  def show_total
+    puts "Total: £#{ money_format(@order.total) }"
+  end
+
+  def show_basket
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for show_basket is too high. [30.02/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [11/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)